examples/typescript/ucloud/main.ts

```diff
@@ -0,0 +1,33 @@
+import { Construct } from 'constructs';
+import { App, TerraformStack } from 'cdktf';
+import { UcloudProvider, Instance, DataUcloudImages } from './.gen/providers/ucloud'
+
+class MyStack extends TerraformStack {
+  constructor(scope: Construct, name: string) {
+    super(scope, name);
+
+    new UcloudProvider(this, 'UCloud', {
+      region: 'cn-bj2',
```

Comment:
It seems like projectId is missing here, judging the failing test

Reply:
Thanks for your reply, provider load the environment variable when I tested it on local, and I just fix it. How can I retry this test case manually?

Reply:
I think your last commit made TS happy 🙌